tests/foreman/cli/test_discoveryrule.py

```diff
@@ -245,9 +245,16 @@ def test_positive_create_with_priority(self):
 
         :CaseImportance: Critical
         """
-        rule_priority = '100'
+        rule_list = DiscoveryRule.list()
+        p_list = []
+        for rule in rule_list:
+            p_list.append(rule['priority'])
+        while True:
```

Comment:
i'd probably use some upper limit for the max number of iterations as a safety fuse

Reply:
thanks for pointing that out
i will limit the no. of iterations